app/controllers/home_page_controller.rb

```diff
@@ -2,5 +2,7 @@ class HomePageController < ApplicationController
   before_action :authenticate_user!
 
   def show
+    @transaction = Transaction.new
+    @recent_records = Transaction.most_recent(current_user)
```

Comment:
@sinpai what do you think about getting transactions from a user?
```ruby
user.rb

scope :most_recent_transactions -> { transactions.most_recent }
```
```ruby
transaction.rb

scope :most_recent, -> { order(created_at: :desc).limit(most_recent_count) }

private

def most_recent_count
  10
end
```

Reply:
@sinpai so, after it you can use
```ruby
@recent_records = current_user.most_recent_transictions
```

Reply:
👍